Gems/ScriptCanvasMultiplayer/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+add_subdirectory(Code)
```

Comment:
This should use the latest Default Gem template which uses a variable to reference the gem name from the `gem.json` file.

It makes it easier to rename the gem by only needing to update the `gem.json` file: https://github.com/o3de/o3de/blob/development/Templates/DefaultGem/Template/CMakeLists.txt#L9-L29

Reply:
Done

Reply:
The Code/CMakeLists.txt also needs to reference the `${gem_name}` like how it does in the [template](https://github.com/o3de/o3de/blob/development/Templates/DefaultGem/Template/Code/CMakeLists.txt#L59)

Reply:
Done.